lms/static/styles/components/_FileList.scss

```diff
@@ -24,17 +22,10 @@ $file-list-icon-size: 24px;
 }
 
 .FileList__icon {
-  margin-left: 10px;
-  margin-right: 10px;
-  width: $file-list-icon-size;
-  height: $file-list-icon-size;
-  vertical-align: middle;
-
-  &.is-selected {
-    // nb. Argument must be passed to `invert` even though "1" is the default
-    // to work around https://bugs.chromium.org/p/chromium/issues/detail?id=964696.
-    filter: #{'invert(1)'}; // Quoted to avoid conflict with SASS mixin.
```

Comment:
To check my understanding, this is replaced by changing the `color` property of the icon to white in the `is-selected` class for file icons in selected rows?

Reply:
Yep. I should have called this out in a comment! Since we're using an inline SVG that takes `currentColor`, there is no need for this: we can style `.is-selected` with basic `color` and `background-color` rules and it will style the SVG correctly without the need for a `filter`.